cli/src/main/kotlin/OrtMain.kt

```diff
@@ -45,6 +46,7 @@ import org.ossreviewtoolkit.commands.*
 import org.ossreviewtoolkit.model.OrtResult
 import org.ossreviewtoolkit.model.config.LicenseFilenamePatterns
 import org.ossreviewtoolkit.model.config.OrtConfiguration
+import org.ossreviewtoolkit.model.readValue
```

Comment:
Consider swapping the order of this commit with the previous one.

Reply:
I liked it better to first refactor the advisor / scanner to have the extracted code at the same location as the other subcommands, and then extract the code.